system/languages/nl.php

```diff
@@ -154,7 +154,7 @@
 		"general-info" => "algemene informatie",
 		"lineup" => "lineup",
 		"timetable" => "blokkenschema",
-		"view" => "view",
+		"view" => "bekijk",
```

Comment:
This one as well (can probably be in a commit together with the other translation fix above)